YouVsVirus/Assets/Scenes/Levelgethome/PlayerHouse.cs

```diff
@@ -56,28 +55,39 @@ void OnTriggerEnter2D(Collider2D other)
                 {
                     //activate player in house, deactivate player, end game
                     StartCoroutine(SetPlayerInHouse(other.gameObject.GetComponentInParent<HumanBase>().gameObject));
-                    // make the toilet paper stop blinking
-                  //  GameObject.Find("ToiletpaperPlayer").GetComponent<SpriteBlink>().startBlinking = false;
+   
                 }
             }
         }
     }
 
+    public void ShowPlayer()
+    {
+        playerRend.sortingLayerName = "Default";
+        playerRend.sortingOrder = 0;
```

Comment:
We can do this much more unity like with:

```suggestion
        playerRend.enabled = true;
```

Thus, no sorting layer trickery is needed.